_posts/2016-10-31-Kickstarters-Project.md

```diff
@@ -0,0 +1,205 @@
+During these 4 weeks they tried to work in these short sprints but after a week or two it became clear this wouldn't be an easy task.
+A few team members already had left the group because they were assigned to projects, which messed up their sprints completely.
+
+During the first 2 weeks the back-end team started with using Spring Data JPA, but soon figured out Spring-boot-starter-data-MongoDB was the better alternative because of the large amount of data being pumped into the DB.
+A lot of time of went into providing Rest documentation that covers all the different calls handled by the application.
+This documentation was created with MockMvc tests, which create code snippets that are easy to use.
+
+On the front-end side they didn't have test cases yet, but nobody in the entire team had ever written front-end tests before.
+Which caused a little bit of a delay, also the webpages didn't seem to be responsive at all.
+Luckily there were some online tutorials at hand on mocking and writing front-end tests.
+The responsiveness issue was sovled by using their own components and CSS code instead of using material design.
+
+When starting their 2 final weeks there were only 3 people left who were able to fully commit to the dev-case.
+During the last week the kickstarters also had to prepare and give a proper introduction to the management of Ordina.
+Showing their newly learned presentation and introduction techniques.
+
+At the very end of the Kickstarters project the application was finished.
+* You are able to watch an overview of the rooms
+* You are able to check if a room is occupied or available
+* You are able to check which sensors are in a room and check their last history
+* You are able to check the sensor history between 2 timestamps
+* You are able to get notifications on your cellphone when certain values are exceeded
+
+
+### possible future changes
+
+Because there wasn't enough time to completely finish the project, this project can still evolve in a lot of ways.
+
+- Later on it could be possible to add roles or users.
+- Easy to add more sensors or rooms
+- Add predictions to the applications for every room.
+
+
+### Lessons learned during the Kickstarters project
+
+* How to be more confident
+* How to introduce yourself in a professional way
+* How to be more assertive
+* You have to keep an eye out for possible changes in your code that encourage clean code
+* How to work better and agile in a team and how to use SCRUM principles
+* Pitfalls and difficulties when using and combining new technologies
+* How to write proper tests (JUnit, Mockito, MvcTests)
+* How to write proper Rest documentation
+
+
+
+## The new JWorks colleagues
+
+<img alt="Axel Bergmans" src="/img/kickstarters/2016/axel-bergmans.jpg">
+<img alt="Matthias Caryn" src="/img/kickstarters/2016/matthias-caryn.jpg">
+<img alt="Madi Dudaeva" src="/img/kickstarters/2016/madi-dudaeva.jpg"> Madi Dudaeva<br>
```

Comment:
Make all the pictures the same size (check other blog posts how we display people there)
Also, Axels picture isnt showing up